src/dags/vergunningen.py

```diff
@@ -59,13 +57,9 @@
     on_failure_callback=get_contact_point_on_failure_callback(dataset_id=dag_id),
 ) as dag:
 
-    # 1. Post message on slack
+    # 1. Post info message on slack
     slack_at_start = MessageOperator(
-        task_id="slack_at_start",
-        http_conn_id="slack",
-        webhook_token=slack_webhook_token,
-        message=f"{SLACK_ICON_START} Starting {dag_id} ({OTAP_ENVIRONMENT})",
-        username="admin",
+        task_id="slack_at_start"
```

Comment:
Tip: I'd recommend leaving the trailing comma after the last parameter, like you did in the `import` in lines 8-15 and in src/dags/grootstedelijke_projecten.py. Doing so here would reduce this from a six (text) line change to a four line change and make the purpose of the change more evident. This applies to almost all the files here, not just this one. It also has the advantage that if an extra parameter is added later, the change would be two lines shorter than it will in this case.

Reply:
Yes, I've added the trailing comma after the new added "slack_at_start" parameter. The black layout replaced the import of common on a one liner since some of the imports where omitted due to the new slackOperator